uncompyle6/parsers/parse36.py

```diff
@@ -33,6 +33,10 @@ def p_36misc(self, args):
         """
         sstmt ::= sstmt RETURN_LAST
 
+        cf_for_iter  ::= _come_froms FOR_ITER
```

Comment:
Oops - this isn't needed (or used) anymore. The decision (for now) is not to add COME_FROMS to backward (in current Python code generation that means looping) jumps.